action.yml

```diff
@@ -139,13 +139,11 @@ runs:
       shell: bash
 
     - name: Enable inline Tutor plugins
-      env:
-        INLINE_TUTOR_PLUGINS_PATH: ${{ inputs.app_name }}/${{ inputs.inline_tutor_plugins_folder }}
       run: |
```

Comment:
Does this work?
```suggestion
      if: ${{ inputs.inline_tutor_plugins_folder }}
      run: |
```

Reply:
We need to enable the `patches` plugin, so we need to execute this step. An alternative would be to have a separate step to enable the plugins. Like this:

```yml
- name: Copy inline Tutor plugins to the plugins directory
  if: ${{ inputs.inline_tutor_plugins_folder }}
  run: |
    source .tutor_venv/bin/activate
  
    cp -r "${{ inputs.app_name }}/${{ inputs.inline_tutor_plugins_folder }}"/* plugins/
  shell: bash

- name: Enable inline Tutor plugins
  run: |
    source .tutor_venv/bin/activate
  
    for plugin_file in plugins/*; do
      plugin_name=$(basename "$plugin_file" | cut -f1 -d '.')
      tutor plugins enable "$plugin_name"
    done
  shell: bash
```
What do you think? @mariajgrimaldi

Reply:
I like the simplicity of leaning into what gh actions offer instead of implementing our own flow control with bash, so thank you for considering the suggestion!